packages/toggle/src/Toggle.tsx

```diff
@@ -43,7 +43,7 @@ const focusStateStyle = css`
   bottom: -2px;
   left: -2px;
   right: -2px;
-  border: 2px solid rgba(67, 177, 229, 0.25);
+  border: 2px solid #63b0d0;
```

Comment:
I think that the mocks have the interaction ring at 3px, rather than the 2 it originally was.

Reply:
👍